db/migrate/20220317133810_create_companies.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateCompanies < ActiveRecord::Migration[7.0]
+  def change
+    create_table :companies do |t|
+      t.string :name
+      t.string :symbol
+
+      t.timestamps
+    end
+  end
```

Comment:
I guess there shouldn't be 2 companies with the same `symbol`, right?

If yes, I would add an index in the database with unique constraint:

```ruby
add_index :companies, :symbol, unique: true
```

Then the database would ensure uniqueness.

Reply:
Yes, companies should be unique, good idea